.github/workflows/ci.yml

```diff
@@ -244,6 +244,55 @@ jobs:
       - name: Run semver-checks
         run: cargo run semver-checks check-release --manifest-path="subject/crates/bevy_gltf/Cargo.toml" --baseline-version 0.9.0
 
+  run-on-clap:
+    # clap v3.2.0 added a semver violation
+    # check if cargo-semver-checks detects the issue
+    # https://github.com/clap-rs/clap/issues/3876
+    name: Run cargo-semver-checks on clap v3.2.0
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout cargo-semver-checks
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Checkout clap v3.2.0
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          repository: 'clap-rs/clap'
+          ref: 'v3.2.0'
+          path: 'subject-current'
+
+      - name: Checkout clap v3.1.18
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          repository: 'clap-rs/clap'
+          ref: 'v3.1.18'
+          path: 'subject-baseline'
+
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: clap
+
+      - name: Run semver-checks
+        run: ! cargo run semver-checks check-release --manifest-path="subject-current/Cargo.toml" --baseline-root="subject-baseline/Cargo.toml" > output
+
+      - name: Check output
+        run: |
+          EXPECTED=$(echo -e "--- failure auto_trait_impl_removed: auto trait no longer implemented ---\n--- failure trait_missing: pub trait removed or renamed ---")
+          RESULT=$(cat output | grep failure)
```

Comment:
Maybe let's also check that no other failures have been found?

Reply:
What do you mean? I check if the failures are the exact ones that are expected.

Reply:
Oh, I see. Thanks for explaining.